bookmarks/tests.py

```diff
@@ -49,6 +49,20 @@ def test_no_user(self):
 
         self.assertEqual(404, resp.status_code)
 
+    def test_bookmarks_oldusername(self):
+        user = User.objects.get(username='Anton')
```

Comment:
An alternative way of testing the redirects would be to:
- get user
- rename
- save (just as you do)
- call page with original username
- call page with new username
- check that content of page with both usernames is the same

Wouldn't that be easier and cleaner?
In the current tests you only check that it works with the old username, but not with the new. If we break the `get_user...` util function for "new" usernames the test would not catch that.

Reply:
Yes, it's true. This is because in general there is another test for the regular case, where the username is not in OldUsername table. But you're right is not exactly the same case.